File: Backend/PortfolioAPI/Models/Entities/Testimonial.cs
namespace PortfolioAPI.Models;

public class Testimonial
{
    public int Id { get; set; }
    public string ClientName { get; set; } = string.Empty;
    public string? ClientNameAr { get; set; }
    public string? ClientTitle { get; set; }
    public string? ClientTitleAr { get; set; }
    public string? Content { get; set; }
    public string? ContentAr { get; set; }
    public int Rating { get; set; } = 5;
    public string? AvatarUrl { get; set; }
    public int SortOrder { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: Frontend/portfolio-app/src/core/types/testimonial.ts
export interface Testimonial {
  id: number;
  clientName: string;
  clientNameAr?: string;
  clientTitle?: string;
  clientTitleAr?: string;
  avatarUrl?: string;
  content: string;
  contentAr?: string;
  rating: number;
  sortOrder: number;
}
